lib/engine/game/g_1849/game.rb

```diff
@@ -436,6 +466,12 @@ def close_corporation(corporation, quiet: false)
           end
           corporation.next_to_par = true if @corporations[index - 1].floated?
           update_garibaldi
+          # code below is for variant rules
+          @log << 'E-token returned to bank' if corporation.e_token
+          corporation.e_token = false if electric_dreams?
```

Comment:
Doesn't the bank's e-token count need to be increased?

Reply:
Also don't need to gate by `if electric_dreams?` here because the attribute always exists on the corporation.

Reply:
removed the `if electric_dreams?` gate. 

The bank doesn't actually have e-tokens, it's just a true/false that I set up. The log is just using the terminology used by the rules.

Commit name: `remove unnecessary electric_dreams? gate`